packages/hint/src/lib/config.ts

```diff
@@ -388,7 +388,7 @@ export class Configuration {
         }
 
         // In case the user uses the --watch flag when running hint
-        if (actions && actions.watch) {
+        if (actions && actions.watch && userConfig.connector && userConfig.connector.options) {
```

Comment:
🤞 that https://github.com/tc39/proposal-optional-chaining will advance stage 1 one day so we can avoid this type of code.